.github/workflows/build.yaml

```diff
@@ -18,14 +18,23 @@ jobs:
     strategy:
       matrix:
         go: ['1.11', '1.12', '1.13', '1.14']
+      # Build all variants regardless of failures
+      fail-fast: false
     name: Go ${{ matrix.go }}
 
     steps:
       - uses: actions/checkout@v2
         with:
-          # Assuming that there are at most 20 commits in a single pull
-          # request, we want to check all commits in a pull request have DCO.
-          fetch-depth: 20
+          # By default, the checkout action only fetches the last commit,
+          # but we want to run DCO check against all commit messages.
+          # Assuming that a pull request may have multiple commits,
+          # fetching the single commit doesn't work for us.
+          #
+          # Since the checkout action automatically creates a merge commit
+          # for merging "master" into a pull request branch, we need to
+          # have 21 commits (including the merge commit) to have
+          # 20 actual commits.
+          fetch-depth: 21
```

Comment:
Can we also state that the 20 comes from the `make lint/git-validation` rule? Just to be more clear

Reply:
Sure. Will do.